Flow.Launcher/SettingWindow.xaml

```diff
@@ -607,7 +607,8 @@
                     <ScrollViewer
                         Margin="0,0,0,0"
                         Background="{DynamicResource Color01B}"
-                        ScrollViewer.CanContentScroll="False">
+                        ScrollViewer.CanContentScroll="False"
```

Comment:
You will need to set CanContentScroll to true to enable virtualization. I will handle that for you